assets/js/base/components/quantity-selector/stories/index.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import { Story, Meta } from '@storybook/react';
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import QuantitySelector, { QuantitySelectorProps } from '..';
+
+export default {
+	title: 'WooCommerce Blocks/@base-components/QuantitySelector',
+	component: QuantitySelector,
+	args: {
+		itemName: 'widgets',
+	},
+	argTypes: {
+		quantity: { control: { disable: true } },
```

Comment:
Why is the control for quantity disabled here? I suggest it might be useful to let a user vary the initial quantity with a control here

Reply:
@alexflorisca I've updated this, but because the way this component is supposed to be used is that the state is updated by its own plus/minus, though it still technically is a controlled component, it doesn't play in the best way with Storybook, and I had to use some magic to make it work.

Notice that, however, because of [this bug](https://github.com/storybookjs/storybook/issues/15924) you can change the number from the control and it will reflected in the component, but not viceversa.